### ISSUE TITLE
Create Debian package

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-proxy#6]
As with other workstation dependencies, we should create a Debian package for the proxy so that it can be installed easily as part of provisioning. See freedomofpress/securedrop-workstation#136 for background and discussion about packaging approaches.

**Comment by redshiftzero** [comment]
Next steps on this are:

- [ ] @redshiftzero (or anyone else available on SD team, we can cycle through everyone on the team so they are familiar with how to package as described in https://github.com/freedomofpress/securedrop-workstation/issues/136#issuecomment-425158072) to run through our packaging workflow [using contents here](https://github.com/freedomofpress/securedrop-debian-packaging) to ensure that the process is clear (we'll discuss and update docs / make changes if needed)
- [ ] @kushaldas to be onboarded to apt-test
- [ ] @kushaldas to push deb package to apt-test
- [ ] we file a followup issue to modify salt logic in the workstation repo to use this package

**Comment by eloquence** [comment]
That PR has since been merged. @kushaldas, could you provide an update on the status of this ticket, please?

**Comment by kushaldas** [comment]
All the packaging related files for `securedrop-proxy` are now under https://github.com/freedomofpress/securedrop-debian-packaging/tree/master/securedrop-proxy/debian directory. As a next step, Jen (or someone else) will try to rebuild the package locally following the guide.

**Comment by redshiftzero** [comment]
Just one remaining task here: @kushaldas can you upload this package to apt-test? Then we can file a followup to use it over on the workstation side

**Comment by kushaldas** [comment]
@redshiftzero Yes, I will do that today (I pinged @emkll yesterday as I may require his help).

**Comment by redshiftzero** [comment]
@kushaldas reports this is blocked on https://github.com/freedomofpress/securedrop-proxy/issues/11

**Comment by redshiftzero** [comment]
Note: this got unblocked on Friday

**Comment by kushaldas** [comment]
@redshiftzero @eloquence Can we move this issue to done?

**Comment by redshiftzero** [comment]
thanks @kushaldas! filed https://github.com/freedomofpress/securedrop-workstation/issues/161 as followup
